x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_table/alerts_sub_grouping.tsx

```diff
@@ -139,7 +143,6 @@ export const GroupedSubLevelComponent: React.FC<AlertsTableComponentProps> = ({
   multiValueFieldsToFlatten,
   pageScope = PageScope.alerts,
   onAggregationsChange,
-  emptyGroupingComponent,
```

Comment:
Was removing this intentional? I see that it does not seem used in this file, but just checking to be sure